src/_locales/zh_CN/messages.json

```diff
@@ -439,6 +439,25 @@
     "hash": "2aff10770796694827ec4253860e0771"
   },
 
+  "translatorCredit": {
+    "message": "该扩展简体中文翻译由 Cye3s 完成",
```

Comment:
You know you need to mention the string `$TRANSLATORS$` here?
See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Retrieving_message_strings_from_JavaScript for a different example.

This is the placeholder, that will automatically be replaced. So don't hardcode your name here, this is what we have `translatorUsername` for (and `translatorLink` for the link).

Generally, you can easily test [the add-on locally](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding) to see if everything is correctly translated. (You may also notice missing translations then.)